hugo/config.toml

```diff
@@ -52,14 +54,14 @@ canonifyURLs = true
     url = "/api"
     weight = -999
   [[menu.main]]
-    identifier = "git"
-    pre = "<i class='icon-github'></i>"
-    name = "Doc Source"
-    url = "https://github.com/ga4gh-discovery/ga4gh-search"
+    identifier = "specs"
+    pre = "<i class='icon-file-text'></i>"
+    name = "Search Specs"
```

Comment:
How about calling this "Specification" and renaming "Open API 3 Reference" to "API"?